src-lib/yolo_layer.cpp

```diff
@@ -32,6 +32,7 @@ layer make_yolo_layer(int batch, int w, int h, int n, int total, int *mask, int
     l.classes = classes;
     l.cost = (float*)xcalloc(1, sizeof(float));
     l.biases = (float*)xcalloc(total * 2, sizeof(float));
+    l.nbiases = total * 2;
```

Comment:
Can you provide some details on this?  I have no idea what it is, where it gets used, why it might have been missing, etc.

Reply:
>  This PR is a request for convenience.
>  It doesn't affect the use of darknet.

Hello,
I'm using darknet integrated with Python.

When the model is loaded in darknet, I'm working on allowing Python to access the model's information.
At this point, I'm sending the structural information of all layers of the model to Python, but when passing the length of the bias pointer of the 'YOLO' layer, the length (nbiases) value is 0.

For the YOLO layer, the bias value has a length of anchor * 2, and it would be great if this information could also be confirmed from nbiases.

Thank you for checking.